examples/oriented_images.js

```diff
@@ -0,0 +1,107 @@
+/* global itowns, document, renderer, initPicture  */
+// # Simple Globe viewer
+
+// Define initial camera position
+var positionOnGlobe = {
+    longitude: 2.423814,
+    latitude: 48.844882,
+    altitude: 100 };
+
+var promises = [];
```

Comment:
I don't think you need to store the `promises`: we don't need to show the layers used in this example since it's not really the point of this example.

Reply:
ok